packages/story-editor/src/utils/idleCallback.ts

```diff
@@ -17,15 +17,15 @@
 /* Source: https://github.com/behnammodi/polyfill/blob/master/window.polyfill.js */
 const requestIdleCallback =
   window.requestIdleCallback ||
-  function (callback, options) {
+  function (callback: IdleRequestCallback, options?: IdleRequestOptions) {
     options = options || {};
     const relaxation = 1;
     const timeout = options.timeout || relaxation;
     const start = performance.now();
     return setTimeout(() => {
       callback({
         get didTimeout() {
-          return options.timeout
+          return options?.timeout
```

Comment:
Since `options = options || {};` then `options` should always exist though, no?

Reply:
But the property of timeout may not

Reply:
Yes, but the change doesn't influence that -- the added `?` has no effect related to the `timeout` prop being undefined or not. And since `options` always exist, it doesn't do anything basically.

Reply:
```suggestion
          return options.timeout
```